### DISCUSSION TITLE
line 4: Bad configuration option: identifyfile

### DISCUSSION THREAD

**Discussion by it-starts-with-e** [community/community#32680] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, 

I'm new to GitHub so I'm in need of some help. I forgot my SSH key password and when I entered it 3 times I ended deleting the SSH key that I had and created a new one and linked it together. Now when verifying it to see if it works after entering ssh -T git@github.com I receive the following:
ernestcezar@Ernests-MacBook-Pro ~ % ssh -T git@github.com
/Users/ernestcezar/.ssh/config: line 4: Bad configuration option: identifyfile
/Users/ernestcezar/.ssh/config: terminating, 1 bad configuration options

I've tried so many times to reset it but keep getting this message. and I don't know what I am doing wrong.

**Comment by airtower-luna** [comment]
Looks like you made a typo in the SSH config file. The error says you have set an invalid option name `identifyfile`. I assume you mean `IdentityFile`.

**Reply by it-starts-with-e** [reply]
Hello, thank you for your response.

So that was the output that I got after entering ssh -T git@github.com

I have checked for misspelling etc. but couldn't see any.

**Reply by airtower-luna** [reply]
I told you *exactly* where the misspelling is above: You wrote identi**f**yfile where you should write Identi**t**yFile.

**Reply by it-starts-with-e** [reply]
But that is my output the output is that

**Reply by airtower-luna** [reply]
Yes. The output quotes the error in the config file.

**Reply by it-starts-with-e** [reply]
Thank you now it makes sense. the whole time I thought it was something I was entering in terminal not in the config file. forgive me my caffeine hasn't kicked in yet lol. Now I have to fix this permission denied (publickey)

**Reply by airtower-luna** [reply]
That either means you haven't added the right key to your GitHub account yet, or SSH isn't using it for some reason. Add `-v` to the SSH command line to see details, including the keys being used.

**Reply by it-starts-with-e** [reply]
Thank you, this was very helpful, I'm also beginning to realize every character and keystroke counts haha

**Comment by JustOptimize** [comment, marked as answer]
In your ssh config file you wrote `identifyfile` instead of `IdentityFile`
The config file is located in `/Users/ernestcezar/.ssh/config` and the error in the 4th line.

As described [here](https://stackoverflow.com/questions/16116788/github-ssh-config-bad-configuration-option-identifyfile).

**Reply by it-starts-with-e** [reply]
Thank you, it wasn't registering in my head that it was something that had to do with the config file itself. I thought it was something that I entered wrong in terminal so I kept on scrolling to see what my keystrokes were. now that is cleared up I have to figure out this permission denied public key *sigh

**Reply by JustOptimize** [reply]
@it-starts-with-e, if this solved your issue you should mark this as your answer or this discussion will always be marked as "Unanswered"
